db/userAddresses.js

```diff
@@ -0,0 +1,46 @@
+const client = require("./client");
+
+async function addShippingAddress({
+    userId,
+    street,
+    city,
+    state,
+    zip
+}) {
+    try{
+        const { rows: [shippingAddress] } = await client.query(`
+            INSERT INTO shippingAddresses("userId", street, city, state, zip)
```

Comment:
Not sure if it makes a difference but it might. The naming convention for tables with multiple words fitnesstrackr used was routine_activities so I think shipping addresses would be shipping_addresses.

Reply:
Done.